### ISSUE TITLE
1.8.0版本，conv_transpose_eltwiseadd_bn_fuse_pass导致PaddleOCR检测模型CPU预测结果错误

### ISSUE THREAD

**Issue by cryoco** [PaddlePaddle/Paddle#24585]
**System information**
-PaddlePaddle version: `v1.8.0`, `0231f58e592ad9f673ac1832d8c495c8ed65d24f`
-CPU: `Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz`
-GPU: V100
-OS Platform (eg.Mac OS 10.14): Ubuntu1404 
-Docker image: `hub.baidubce.com/paddlepaddle/paddle:latest-dev`
-Python version: Python3.7
-Cmake orders
-C++version.txt: used python api
-API information: inference configuration
```python
config.switch_use_feed_fetch_ops(False)
config.disable_gpu()
```

**To Reproduce** 

1. run test with `conv_transpose_eltwiseadd_bn_fuse_pass`

``` shell
python3.7 test_ocr_det.py --model_file=det/model --params_file=det/params --float32
```

The head part of output：
![image](https://user-images.githubusercontent.com/12407750/82036396-709fad00-96d3-11ea-8793-0194dae32297.png)

2. run test without `conv_transpose_eltwiseadd_bn_fuse_pass`
by adding 
`config.delete_pass("conv_transpose_eltwiseadd_bn_fuse_pass")` in AnalysisConfig.

``` shell
python3.7 test_ocr_det.py --model_file=det/model --params_file=det/params --float32
```

The head part of output：
![image](https://user-images.githubusercontent.com/12407750/82036641-b8becf80-96d3-11ea-9b2d-0f37c0b32fa3.png)

**Describe your current behavior**
It has been verified that when using `conv_transpose_eltwiseadd_bn_fuse_pass`, the cpu inference output is wrong. It's easy to check the right output by using GPU to perform inference:

```python
#config.disable_gpu()
config.enable_use_gpu(100, 0)
```

This issue can be reproduced with and without MKLDNN.

**Code to reproduce the issue**
[test_ocr_det.txt](https://github.com/PaddlePaddle/Paddle/files/4633635/test_ocr_det.txt)

Need to be renamed to test_ocr_det.py

**Model to reproduce the issue**

```shell
wget https://paddleocr.bj.bcebos.com/inference.tar
tar -xf inference.tar
```
The `det` model is the one which reproduces this issue.

**Other info / logs**

**Comment by luotao1** [comment]
@jczaja @lidanqing-intel Could you help see it?

**Comment by lidanqing-vv** [comment]
> @jczaja @lidanqing-intel Could you help see it?

Ok, we will investigate this.

**Comment by jczaja** [comment]
@cryoco , @luotao1  Is this problem reproducible at PP 1.7 ?

**Comment by cryoco** [comment]
> @cryoco , @luotao1 Is this problem reproducible at PP 1.7 ?

@jczaja Yes, it happens at v1.7.2 too.

**Comment by jczaja** [comment]
@luotao1 , @cryoco I just wanted to know that We have reproduced issue and are currently testing fix , which is here: #24756.

**Comment by jczaja** [comment]
@cryoco  PR #24756 (merged) introduced fix to this issue. Please check (using recent develop) if issue is solved. and let us know

**Comment by lidanqing-vv** [comment]
Hi, @luotao1  this issue does not exist anymore on our local machine because of merged PR #24756  Could Baidu validation team check if we can close this issue ? Thanks

**Comment by cryoco** [comment]
@jczaja @lidanqing-intel Sorry for the late reply. We confirmed that this issue is solved on latest develop. Thanks!!
